h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstSum.java

```diff
@@ -5,15 +5,15 @@
 /**
  */
 public class AstSum extends AstRollupOp {
-  public String str() {
-    return "sum";
-  }
+    public String str() {
+        return "sum";
+    }
 
-  public double op(double l, double r) {
-    return l + r;
-  }
+    public double op(double l, double r) {
+        return l + r;
+    }
 
-  public double rup(Vec vec) {
-    return vec.mean() * vec.length();
-  }
-}
+    public double rup(Vec vec) {
+        return vec.mean() * vec.length();
+    }
+}
```

Comment:
Could you please undo changes to this file? As far as I remember we have a style rule that java files use 2-spaces indent...